### PR TITLE
add documentation for 'stat'

### LINKED ISSUE #281: add docs for `stat` command
Command was mentioned in lecture but is not defined on our website.

### PR DESCRIPTION
**Pull request by umjaffer**
# Advanced HW 5 submission

Closes #281

Name: Usmaan Jaffer
Uniqname: ujaffer

### CODE REVIEW

File: _commands/tools/stat.md [hunk 1]
```diff
@@ -3,22 +3,53 @@
...
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
+The output is structured as below:
+~~~ bash
+st_mode st_ino st_dev st_rdev st_nlink st_uid st_gid st_size st_atim st_mtim st_ctim st_blksize st_blocks
+~~~
+`st_dev` – identifier of device containing file
```
Comment (AmritHariharan) [inline _commands/tools/stat.md]: this doesn't seem to be showing up properly as a list

**Comment by AmritHariharan** [comment]
Some markdown formatting issues, commented on file.

**Comment by umjaffer** [comment]
looks all good now

**Comment by cyanliu** [comment]
You've been awarded credit for advanced homework 5 and given 2 points of extra credit for creating your initial PR more than 2 days before the deadline 🎉 

You won't see this immediately reflected on Canvas, but it will be there the next time we update grades!

**Pull request merged**